tests/tests/abi_tests.cpp

```diff
@@ -354,23 +354,24 @@ BOOST_AUTO_TEST_CASE(uint_types)
 
 } FC_LOG_AND_RETHROW() }
 
+struct abi_gen_helper {
 
-BOOST_AUTO_TEST_CASE(generator)
-{ try {
+  const char* eosiolib_path = nullptr;
 
-  if( std::getenv("EOSLIB") == nullptr ) {
-    wlog("*************************************");
-    wlog("*  EOSLIB env variable not defined  *");
-    wlog("*  ABIGenerator tests will not run  *");
-    wlog("*************************************");
-    return;
+  abi_gen_helper() {
+    eosiolib_path = std::getenv("EOSLIB");
```

Comment:
Instead of relying on a soft link at invocation time, can we have a `config.h.in` file and use cmake to properly set the path?  Is there a time when we will build in one tree and run the tests on that code but want to run abi_generator tests on another tree?

Reply:
I feel like the abi_generator meat needs to be refactored in to a library that can be used here instead.

Reply:
Though that doesn't help with the include file dependency...